src/snap/grpc_export.cc

```diff
@@ -47,6 +48,40 @@ using json = nlohmann::json;
 
 using Plugin::PluginInterface;
 using Plugin::Meta;
+using Plugin::PluginException;
+
+bool file_exists(const std::string path) {
```

Comment:
Should be defined as a private method in the header. Otherwise this will be exposed to outside use.